<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '0', '1', '1', '1', '0', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '0', '1', '1', '1', '0', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '0', '1', '1', '1', '0', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '0', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '0', '0', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '0', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 57, in <module>
    mC = frontend.EnsembleMetricsCalculator(expe_config, 'distance_metrics')
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 40, in __init__
    super().__init__(expe_config)
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/configurate_ens.py", line 208, in __init__
    os.mkdir(self.log_dir)
FileNotFoundError: [Errno 2] No such file or directory: "/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0']/log/"
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 57, in <module>
    mC = frontend.EnsembleMetricsCalculator(expe_config, 'distance_metrics')
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 40, in __init__
    super().__init__(expe_config)
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/configurate_ens.py", line 208, in __init__
    os.mkdir(self.log_dir)
FileNotFoundError: [Errno 2] No such file or directory: "/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1']/log/"
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '1', '1', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '0', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '1', '0', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '0', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '0', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '1', '1', '0', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '0', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '0', '1', '1', '1', '1', '0', '0', '1', '1', '1', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '0', '1', '1', '1', '1', '0', '0', '1', '1', '1', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '0', '1', '1', '1', '1', '0', '0', '1', '1', '1', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '1', '0', '1', '0', '0', '1', '1',''1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '0', '1', '0', '1', '0', '0', '1', '1',''1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '1', '0', '1', '0', '0', '1', '1',''1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 57, in <module>
    mC = frontend.EnsembleMetricsCalculator(expe_config, 'distance_metrics')
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 40, in __init__
    super().__init__(expe_config)
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/configurate_ens.py", line 208, in __init__
    os.mkdir(self.log_dir)
FileNotFoundError: [Errno 2] No such file or directory: "/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '1', '0', '1', '0', '0', '1', '1',''1']/log/"
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '0', '0', '1', '1', '1', '0', '1', '1', '0', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '0', '0', '1', '1', '1', '0', '1', '1', '0', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '0', '0', '1', '1', '1', '0', '1', '1', '0', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '1', '1', '0', '1', '1', '1', '1', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '0', '1', '1', '0', '1', '1', '1', '1', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '0', '1', '1', '0', '1', '1', '1', '1', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '0', '0', '1', '1', '1', '1', '1', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '1', '0', '0', '1', '1', '1', '1', '1', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '0', '0', '1', '1', '1', '1', '1', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '0', '1', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '0', '1', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/normal_['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '0', '1', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '0', '0', '0', '1', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '1', '0', '0', '0', '1', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '0', '0', '0', '1', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '0', '0', '1', '0', '1', '0', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '0', '0', '1', '0', '1', '0', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '0', '0', '1', '0', '1', '0', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '1', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '1', '1', '1', '0', '0', '0', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '1', '0', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '1', '0', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '1', '0', '1', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', '0', '1', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', '0', '1', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', '0', '1', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['1', '0', '0', '0', '0', '0', '1', '0', '1', '0', '0', '0', '1', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['1', '0', '0', '0', '0', '0', '1', '0', '1', '0', '0', '0', '1', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['1', '0', '0', '0', '0', '0', '1', '0', '1', '0', '0', '0', '1', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '1', '0', '0', '0', '0', '0', '1', '1', '0', '0', '0', '0', '0']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '1', '0', '0', '0', '0', '0', '1', '1', '0', '0', '0', '0', '0']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '1', '0', '0', '0', '0', '0', '1', '1', '0', '0', '0', '0', '0']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '0', '1', '0', '0', '0', '1', '1', '0', '1', '1', '0', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '0', '0', '1', '0', '0', '0', '1', '1', '0', '1', '1', '0', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '0', '0', '1', '0', '0', '0', '1', '1', '0', '1', '1', '0', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['1', '0', '0', '0', '0', '0', '1', '0', '1', '0', '1', '1', '0', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['1', '0', '0', '0', '0', '0', '1', '0', '1', '0', '1', '1', '0', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['1', '0', '0', '0', '0', '0', '1', '0', '1', '0', '1', '1', '0', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
<class 'list'> instance_num
<class 'list'> param_sets
Namespace(data_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '1', '1', '1', '1', '0', '0', '0', '0', '0', '1', '0', '1', '1']/samples/"], data_dir_obs='/scratch/mrmn/moldovang/datasets/obs_full_npy/', data_dir_real='/scratch/mrmn/brochetc/GAN_2D/datasets_full_indexing/IS_1_1.0_0_0_0_0_0_256_large_lt_done/', debiasing=True, expe_name="random_['0', '1', '1', '1', '1', '0', '0', '0', '0', '0', '1', '0', '1', '1']", instance_num=[], length=1, list_steps=[[0]], log_dir_names=["/scratch/mrmn/moldovang/tests_CGAN/random_['0', '1', '1', '1', '1', '0', '0', '0', '0', '0', '1', '0', '1', '1']/log/"], num_proc=8, parameter_sets=[], short_names=['00'], subsample=112, variables=['u', 'v', 't2m'])
[180, 436, 500, 756]
variables ['u', 'v', 't2m']
[180, 436, 500, 756]
Indices of selected variables in samples (real/fake) : [1, 2, 3] [0, 1, 2]
[]
Traceback (most recent call last):
  File "metric_test_exec_ensemble.py", line 60, in <module>
    mC.estimation(distance_metrics_list, program, standalone=False, parallel=False,
  File "/home/mrmn/moldovang/score_ensemble/score_ensemble/evaluation_frontend_ens.py", line 154, in estimation
    print(caca)
NameError: name 'caca' is not defined
